hrms/overrides/expense_claim.py

```diff
@@ -0,0 +1,481 @@
+from hrms.hr.doctype.expense_claim.expense_claim import ExpenseClaim
+from erpnext.accounts.doctype.payment_entry.payment_entry import (
+    get_party_details,
+    get_account_details,
+)
+from erpnext.accounts.general_ledger import make_gl_entries
+from erpnext.accounts.doctype.sales_invoice.sales_invoice import get_bank_cash_account
+from suite42_erp_app.suite42_erp_app.doctype.suite42_application_config.suite42_application_config import (
+    Suite42ApplicationConfig,
+)
+from suite42_erp_app.suite42_erp_app.common.utils import user_has_role
+from suite42_erp_app.suite42_erp_app.common.authentication.role import (
+    RoleConstants,
+    ExpenseClaimConstants,
+    ExpenseCategoryConstants,
+)
+from suite42_erp_app.overrides.custom_task import CustomTask, TaskType
+from suite42_erp_app.suite42_erp_app.common.decorators import (
+    handle_exceptions_with_readable_message,
+)
+from frappe.query_builder.functions import Sum
+from frappe.utils import cstr, flt, get_link_to_form
+import frappe
+from frappe import _
+import json
+
+
+class CustomExpenseClaim(ExpenseClaim):
+    def validate(self):
+        self.validate_approver()
+        self.validate_sanctioned_amount()
+        self.calculate_total_amount()
+        self.validate_advances()
+        if self.expense_category in [
```

Comment:
add expense_claim_flow in expense category and fetch the value and compare

Reply:
Done